Tasks/KubernetesV1/make.json

```diff
@@ -27,11 +27,11 @@
 	"rm": [
         {
             "items": [
-                "node_modules/utility-common/node_modules/vsts-task-lib",
-				"node_modules/docker-common/node_modules/vsts-task-lib",
+                "node_modules/utility-common-v2/node_modules/azure-pipelines-task-lib",
+				"node_modules/docker-common-v2/node_modules/azure-pipelines-task-lib",
 				"node_modules/vsts-task-tool-lib/node_modules/vsts-task-lib",
-				"node_modules/azure-arm-rest/node_modules/vsts-task-lib",
-				"node_modules/kubernetes-common/node_modules/vsts-task-lib"
+				"node_modules/azure-arm-rest-v2/node_modules/azure-pipelines-task-lib",
+				"node_modules/kubernetes-common-v2/node_modules/azure-pipelines-task-lib"
```

Comment:
NIT: indent?

Reply:
will fix in a subsequent check-in. thnx.